ethereum/core/src/main/java/org/hyperledger/besu/ethereum/GasLimitCalculator.java

```diff
@@ -16,13 +16,15 @@
 
 public interface GasLimitCalculator {
 
+  static final long DATA_GAS_LIMIT = 786432;
```

Comment:
Is this a duplicate of MAX_DATA_GAS_PER_BLOCK?

Maybe out of scope of this PR, but does anyone think there's value in having all the specified constants grouped together in one place so they as easily searchable/comparable to the spec?

Reply:
Yes it is a duplicate, but could be fixable with the right refactor. That limit is a per-block concern, so likely does not belong in this class, but rather CancunTargetingGasLimitCalculator instead.